client/views/room/providers/ToolboxProvider.tsx

```diff
@@ -78,19 +78,17 @@ const ToolboxProvider = ({ children, room }: { children: ReactNode; room: IRoom
 
 	const openUserInfo = useCallback(
 		(username) => {
-			switch (room.t) {
-				case 'l':
-					open('room-info', username);
-					break;
-				case 'd':
-					open('user-info', username);
-					break;
-				default:
-					open('members-list', username);
-					break;
+			if (room.t === 'l') {
```

Comment:
any reason for changing this from switch to elseif? 👀

Reply:
Because i wanted to add team members info. But teams cannot be distinguished using `room.t`. Have a look at 85 th line.

Reply:
You can achieve the same with a `switch (true)` but I understand your motive here :) thanks!